server/routes/index.js

```diff
@@ -1290,4 +1292,73 @@ router.route('/charts/new')
       return res.status(500).send({ message: err.message })
     }
 })
+
+/**
+ * Study Details
+ */
+router.route('/study-details')
+  .get(ensureAuthenticated, async (req, res) => {
+    try {
+      const { display_name, role, icon } = req.session;
+      return res.status(200).send(studyDetailsPage({
+        uid: req.user,
+        name: display_name,
+        role,
+        icon
+      }))
+    } catch (error) {
+      console.error(error.message)
+      return res.status(500).send({ message: err.message})
+    }
+  })
+router.route('/api/v1/study-details/all')
+  .get(ensureAuthenticated, async(req, res) => {
+    try {
+      const studyDetails = await mongoData.collection(collections.studyDetails)
+      .find({ owner: req.user }).toArray()
+      return res.status(200).json({ studyDetails })
```

Comment:
Similar to my comment below about consistent response shape we could use `data` (or something else) here.

```suggestion
      return res.status(200).json({ data: studyDetails })
```

Reply:
[1731190](https://github.com/AMP-SCZ/dpdash/pull/81/commits/1731190bd6bcc432277da2d2bd9279ea7b81ea5c)